source/catalog.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="index.html" title="Перейти на главную страницу">
+        <img src="img/logo.svg" alt="Логотип Седоны." width="185" height="96" >
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list nav-list">
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="#">
+                Главная
+              </a>
+            </li>
+            <li class="nav-list__item--current">
+              <a class="nav-list__link" href="#">
+                Фото и видео
+              </a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="#">
+                Форма отзыва
+              </a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="https://htmlacademy.ru/">
+                HTML Academy
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <main class="catalog-container">
+      <h1 class="visually-hidden">Седона</h1>
+      <div class="hero-block-container">
+        <img class="hero-block-container__logo" src="#"  alt="Логотип-приветствие.">
+      </div>
+      <section class="header--gallary">
+        <h2 class="header__title">Фото и видео</h2>
+        <p class="header__description">
+          Не можете решиться на путешествие из-за курса? Фотографии помогут вам забыть о политике и экономике.
+        </p>
+      </section>
+
+      <section class="card">
+        <h2 class="visually-hidden">Каталог фотографий</h2>
+        <ul class="card__list">
+          <li class="card__item">
+            <a class="card__link">
+              <img class="card__photo" src="#" alt="Неродные просторы." width="280" height="280">
+            </a>
+            <h3 class="card__title">Неродные просторы</h3>
+            <p class="card__author">Автор фото: Борис</p>
+            <div class="card__like like">
+            <button class="like-button" type="button">
+              <span class="visually-hidden">Мне нравится</span>
+            </button>
+            <span class="like-counter">1350</span>
+            </div>
```

Comment:
Тут у тебя есть блок like, стоит учесть это для кнопки и счетчика
```suggestion
            <div class="card__like like">
            <button class="like__button" type="button">
              <span class="visually-hidden">Мне нравится</span>
            </button>
            <span class="like__counter">1350</span>
            </div>
```